test_conformance/math_brute_force/main.cpp

```diff
@@ -540,7 +546,17 @@ static int ParseArgs(int argc, const char **argv)
              gWimpyReductionFactor);
     }
 
-    if (singleThreaded) SetThreadCount(1);
+    if (singleThreaded)
+    {
+        vlog("*** WARNING: Reduce worker threads to 1                 ***\n");
+        SetThreadCount(1);
+    }
+    else if (forcedWorkerThreads > 0)
+    {
+        vlog("*** WARNING: Reduce worker threads to %d                ***\n",
```

Comment:
Why does the warning say reduce instead of force?

Reply:
true better will be use force, thanks for your comment, changed